docs/source/community/index.rst

```diff
@@ -0,0 +1,10 @@
+:orphan:
```

Comment:
Is this a new page?

Reply:
yes, it will include all of the community related pages, like this: https://docs-preview.pytorch.org/pytorch/pytorch/149331/community/index.html

Reply:
you click this link (which will become live after this PR is merged): 

<img width="585" alt="Screenshot 2025-04-10 at 11 12 33 AM" src="https://github.com/user-attachments/assets/25052bc2-d472-470b-8d7c-cb54dda153a1" />

Reply:
So many menu bars...